### PR TITLE
memo: init at 0.2 + changed my email address

### PR DESCRIPTION
**Pull request by mrVanDalo**
###### Motivation for this change

I want one of my apps in the nixpkgs upstream
I also want to update my email address in the maintainers list ( separate commit of course )

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
You have created this pull request from a different branch. Please base your pull request on master and add your commits on top.

**Comment by mrVanDalo** [comment]
@Mic92 Sure, I did like described here. https://nixos.org/nixpkgs/manual/#idm140737317305456 (maybe the documentation is out of date?) it says I should not work in the master branch. (maybe misread that, and thought I should not branch from the master)

File: lib/maintainers.nix [hunk 1]
```diff
@@ -483,7 +483,7 @@
   oxij = "Jan Malakhovski <oxij@oxij.org>";
   paholg = "Paho Lurie-Gregg <paho@paholg.com>";
   pakhfn = "Fedor Pakhomov <pakhfn@gmail.com>";
-  palo = "Ingolf Wanger <palipalo9@googlemail.com>";
+  palo = "Ingolf Wanger <nixos@ingolf-wagner.de>";
```
Comment (zimbatm) [inline lib/maintainers.nix]: is that you? the key should be your github handle

Comment (mrVanDalo) [reply lib/maintainers.nix]: Right, damn. both emails belong to me, but the googlemail.com is my github handle. let me check if I can change that

**Comment by mrVanDalo** [comment]
@zimbatm contact@ingolf-wagner.de is my current github email address.

**Comment by zimbatm** [comment]
sorry I meant `palo` should be `mrVanDalo` so that it maps to your github handle. This lets people more easily @mention you if there are issues with these packages.

**Comment by mrVanDalo** [comment]
@zimbatm done. for this I also had to touch all packages I "maintain".

**Comment by zimbatm** [comment]
all good, thanks!

**Pull request merged**